### DISCUSSION TITLE
Autosave edit data

### DISCUSSION THREAD

**Discussion by demeus** [livewire/livewire#3342] [1. Help] [answered]
Is there any way to save form data after specific time?
before i used thhis code, but maybe it can be done using livewire
`        setInterval(function(){
            autoSave();
        }, 120000);
        function autoSave(){
          // save form
        }`

**Comment by ph7jack** [comment, marked as answer]
You can use a **wire:poll**

```
<div wire:poll="save">
    ... form
</div>
```

https://laravel-livewire.com/docs/2.x/polling

**Reply by demeus** [reply]
Great, thank you
